apex-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -50,6 +50,12 @@
       </includes>
     </fileSet>
     <fileSet filtered="true" encoding="UTF-8">
+      <directory>src/main/resources</directory>
+      <includes>
+        <include>**/log4j.props</include>
```

Comment:
Sorry for confusion, but when I renamed the file to log4j.properties everything worked well with Apex. For further testing I created datatorrent-rts installer. Now event the "apex" script tries to use new log4j.properties file. Previously it used to use default log4j.properties shipped by hadoop and use CONSOLE appender as per defaults there. But now as it picks our new log4j.properties it tries to load APEXRFA appender and tries to write in some log file which is throwing exception.

For clarity and simplicity I believe better we have different log file name may be like apexlog4j.properties. @davidyan74 @sanjaypujare  please comment on this issue.

I will change log4j name again once I hear back your comments.